package.json

```diff
@@ -6,29 +6,29 @@
   "scripts": {
     "lint": "eslint .",
     "migrate:local": "knex migrate:latest --env $SERVICE_ENVIRONMENT --knexfile ./src/dao/knexfile.js",
-    "docker:start": "docker-compose run windbreaker-hooks",
+    "docker:start": "docker-compose up windbreaker-hooks",
```

Comment:
Changed this to `up` because `run` will ignore environment variables in docker-compose by default.